Easel/Content/Builder/FontContent.cs

```diff
@@ -1,19 +1,23 @@
 using System.IO;
+using Easel.GUI;
 
 namespace Easel.Content.Builder;
 
 public struct FontContent : IContentType
 {
     public string Path { get; set; }
+
+    public FontOptions FontOptions;
     
     public FontContent() { }
 
-    public FontContent(string path)
+    public FontContent(string path, FontOptions? options = null)
```

Comment:
```suggestion
    public FontContent(string path) { ... }
    public FontContent(string path, FontOptions options) { ... }
```

Please use overloads instead of nullable structs.